templates/strapdown.html.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<body>
+<xmp theme="cerulean" style="display:none;">
```

Comment:
actually, [because `<xmp>` is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/xmp), and this is simply a tag, can we change this to something more modern i.e. `<div>` or `<section>`? and keep the same theme and style attributes? It shouldn't affect the output unless the CSS relies specifically on the `xmp` element.

Reply:
The script is hard coded to look for the xmp tag.

Reply:
This makes me sad :/